test/unit/event.js

```diff
@@ -2678,6 +2678,43 @@ test( ".off() removes the expando when there's no more data", function() {
 	}
 });
 
+test( "preventDefault() on focusin does not throw exception", function( assert ) {
+	expect( 1 );
+
+	var done = assert.async(),
+		input = jQuery( "<input>" ).appendTo( "#form" );
+		input
+			.on( "focusin", function( event ) {
+				var flawlessExecution = true;
+
+				try {
+					event.preventDefault();
+				} catch( theException ) {
```

Comment:
Missing space before `(`. Also, what would you think about capturing the exception and asserting strict equality with `undefined` (for a more valuable diff than `Expected: true Result: false` if this ever fails)?